src/chains/ethereum/ethereum/tests/api/eth/call.test.ts

```diff
@@ -909,10 +910,13 @@ describe("api", () => {
         });
 
         const getDbData = async (trie: GanacheTrie) => {
-          let dbData: (string | Buffer)[] = [];
-          for await (const data of trie.db._leveldb.createReadStream()) {
-            dbData.push(data);
-          }
+          const dbData: [Buffer, Buffer][] = [];
+          const stream = trie.createReadStream();
+          // @ts-ignore TODO: why is this necessary? seems like a bug on ejs' end
+          stream.on("data", ({ key, value }) => {
```

Comment:
I suspect it's because of the conditional export in readable-stream is muddling the types. How about this in the meantime?
```suggestion
          (stream as any).on("data", ({ key, value }) => {
```